src/util/string_utils.js

```diff
@@ -144,6 +144,14 @@ function escape_reg_exp(str) {
     return str.replace(escape_regexp, '\\$&');
 }
 
+// indexOfEnd is the index of of string_to_search including it
+// Based on this: https://stackoverflow.com/a/18893403
+// Didn't use it as is since it creates eslint error: no-extend-native error
+function index_of_end(original_string, string_to_search) {
+    const index = original_string.indexOf(string_to_search);
+    return index === -1 ? -1 : index + string_to_search.length;
+}
+
```

Comment:
I suggest to rename it to `index_after`, add types, and shorten variables -

```suggestion
/**
 * Search the string `s` in `str` and if found return the offset right after it.
 * @type {string} str
 * @type {string} s
 * @returns {number}
 */
function index_after(str, s) {
    const i = str.indexOf(s);
    return i >= 0 ? i + s.length : i;
}

```

Reply:
I like meaningful names (and they are easier to search when needed).
I also attached the link to the original code because it helps to understand the context (i would imagine that this kind of function would be built-in...).

Reply:
IMHO this is a function that is just trivial - I can't think of anyone going to debug it after it being written, ever. Also I don't try to predict what will be added to the js language. Feel free to keep long names if you find it clearer but in trivial cases I tend to prefer short names (i for index, s for string are just standard).